src/cosmoz-bottom-bar-focused-context.js

```diff
@@ -0,0 +1,26 @@
+import { createContext, useContext, useState, useMemo } from '@pionjs/pion';
+import { noop } from '@neovici/cosmoz-utils/function';
+
+export const bottomBarFocused = createContext({
+	bars: [],
+	add: noop,
+	remove: noop,
+});
+customElements.define('bottom-bar-focused-provider', bottomBarFocused.Provider);
+
+export const useBottomBarFocusedCtx = () => useContext(bottomBarFocused) ?? {};
```

Comment:
We don't need this.

Reply:
What is it we don't need? It doesn't work without `export const useBottomBarFocusedCtx = () => useContext(bottomBarFocused) ?? {};`?